build/package/Dockerfile.builder

```
@@ -1,2 +1,2 @@
-FROM golang:1.21.0-alpine
+FROM golang:1.21.7-alpine
```

when building the easegress docker image, we are always using the latest version of Go,.

```suggestion
FROM golang:1.22.0-alpine
```